parsons/google/google_bigquery.py

```diff
@@ -50,18 +95,18 @@ def __init__(self, app_creds=None, project=None, location=None):
         # This attribute will be used to hold the client once we have created it.
         self._client = None
 
+        self._dbapi = dbapi
+
         self.dialect = 'bigquery'
 
-    def copy(self, table_obj, dataset_name, table_name, if_exists='fail',
+    def copy(self, table_obj, table_name, if_exists='fail',
```

Comment:
Any breaking changes concerns?

Reply:
Yes. This will break anyone who has been using BigQuery. I will definitely flag in the release notes. I couldn't think of a great way to solve the problem where the DB Sync expects the copy args to be `table_obj`, `table_name` (in that order).

I am happy to revisit if you think it is going to be too big of an impact.